src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -4393,6 +4457,114 @@ partial class C
                 Diagnostic(ErrorCode.ERR_MemberNameSameAsType, "C").WithArguments("C").WithLocation(4, 18));
         }
 
+        [Fact]
+        public void Constructor_BlockToExpressionBody()
+        {
```

Comment:
>{ [](start = 8, length = 1)

I'd also add test of constructors with initializers:

public D(int value) : base(value) => ...
public D(int value) : base(value) { ... }